assignments/assignment-8/assignment8.1.index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment 8.1</title>
+    </head>
+    <body>
+        <h1>Register for Newsletter</h1>
+        <label for="userNameInput">User name</label>
+        <input
+            type="text"
+            placeholder="User Name"
+            id="userNameInput"
+            name="fullName"
+            required
+        />
+
+        <label for="ageInput">Age</label>
+        <input
+            type="number"
+            id="ageInput"
+            name="age"
+            min="12"
+            max="65"
+            required
+        /><br /><br />
+
+        <label for="emailInput">Email</label>
+        <input
+            type="email"
+            placeholder="Email"
+            id="emailInput"
+            name="email"
+            required
+        />
+        <h2>Preferred Topics</h2>
+        <input
+            type="checkbox"
+            id="sportsCheckboxInput"
+            name="sports"
+            value="sports"
+        />
+        <label for="sportsCheckboxInput">Sports</label><br />
+
+        <input
+            type="checkbox"
+            id="technologyCheckboxInput"
+            name="technology"
+            value="technology"
+        />
+        <label for="technologyCheckboxInput">Technology</label><br />
+
+        <input
+            type="checkbox"
+            id="travelCheckboxInput"
+            name="travel"
+            value="travel"
+        />
+        <label for="travekCheckboxInput">Travel</label><br />
+
+        <h3>Select Frequency Newsletter</h3>
+
+        <input
+            type="radio"
+            id="dailyRadioInput"
+            name="newsletter"
+            value="daily"
+        />
+        <label for="dailyRadioInput">Daily</label><br />
+
+        <input
+            type="radio"
+            id="weeklyRadioInput"
+            name="newsletter"
+            value="weekly"
+        />
+        <label for="weeklyRadioInput">Weekly</label><br />
+
+        <input
+            type="radio"
+            id="monthlyRadioInput"
+            name="newsletter"
+            value="monthly"
+        />
+        <label for="monthlyRadioInput">Monthly</label><br />
+        <br />
+
+        <label for="emailFormatSelect">Select preferred email format</label>
+        <select name="emailFormat" id="emailFormatSelect">
```

Comment:
For select name you can use "preferredEmailFormat"

Reply:
Thank you i fixed